Fotogjengen/hilfling-frontend
# Feature/taheera/archiveboss add new categories places and categories

Added additional functionalities for adding new album, category and place. 

How to test:
1. Go to `/intern/arkivsjef`
2. Press the "Add" button at the top of the page
3.  Create either new album, place and category
4. Check if the items in the menu is updated
